pyspectral/avhrr_rsr.py

```diff
@@ -88,12 +88,14 @@ def _load(self, scale=1.0):
 
 def main():
     """Main"""
-    for platform_name in ["NOAA-17", "NOAA-16", "NOAA-14", "NOAA-12",
```

Comment:
Why is this being removed ?

Reply:
Ok, this is just a main script to produce the hdf5 RSR files from the original. It will normally only be used once, when we/I generate the hdf5 files